Makefile

```diff
@@ -1,6 +1,6 @@
 CC = cc
 NAME = minishell
-COMPILE_FLAGS = -Wall -Wextra -Werror -g -c
+COMPILE_FLAGS = -Wall -Wextra -Werror -g -c -DDEBUG
```

Comment:
don't do it in makefile globally, just define it in the test file you're debugging like [here](https://github.com/ioannnikulin/minishell/blob/dev/libft/tests/tree/ft_treenode_insert_child_before_first_test.c#L14)